src/pybind/mgr/dashboard/ci/cephadm/start-cluster.sh

```diff
@@ -63,6 +63,8 @@ cd ${CEPH_DEV_FOLDER}
 : ${VM_IMAGE_URL:='https://download.fedoraproject.org/pub/fedora/linux/releases/36/Cloud/x86_64/images/Fedora-Cloud-Base-36-1.5.x86_64.qcow2'}
 kcli download image -p ceph-dashboard -u ${VM_IMAGE_URL} ${VM_IMAGE}
 kcli delete plan -y ceph || true
+# Compile cephadm locally for the shared_ceph_folder to pick it up
+${CEPH_DEV_FOLDER}/src/cephadm/build.by ${CEPH_DEV_FOLDER}/src/cephadm/cephadm
```

Comment:
we are trying to execute a python file here
```suggestion
exec python3 ${CEPH_DEV_FOLDER}/src/cephadm/build.by ${CEPH_DEV_FOLDER}/src/cephadm/cephadm
```